dump25/fix_parts.py

```
@@ -0,0 +1,127 @@
+                yield line
+
+    def read_file(self, parts_dir):
+        jsonfiles = list(parts_dir.glob("*.json"))
+        print(f"all json files: {len(jsonfiles)}")
+        # ---
+        cc = 1
+        # ---
+        for x in tqdm.tqdm(jsonfiles):
+            lines = self.get_lines(x)
+            # ---
+            for cc, line in tqdm.tqdm(enumerate(lines, start=cc)):
```

Ruff (0.8.2)

Loop control variable `cc` overrides iterable it iterates

(B020)